From abd6591fb5cc1490ab2cf5d9ff513a6677fa57b2 Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Wed, 29 Jul 2026 10:20:43 -0700
Subject: [PATCH 13349/15333] ubifs: key.h: use correct function parameter name

Use @hash as the function parameter name to prevent kernel-doc warnings:

Warning: fs/ubifs/key.h:42 function parameter 'hash' not described in 'key_mask_hash'
Warning: fs/ubifs/key.h:42 Excess function parameter 'val' description in 'key_mask_hash'

Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Reviewed-by: Zhihao Cheng <chengzhihao1@huawei.com>
Signed-off-by: Richard Weinberger <richard@nod.at>
---
 fs/ubifs/key.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/ubifs/key.h b/fs/ubifs/key.h
index 8142d9d6fe5d..71497c1b4753 100644
--- a/fs/ubifs/key.h
+++ b/fs/ubifs/key.h
@@ -33,7 +33,7 @@
 
 /**
  * key_mask_hash - mask a valid hash value.
- * @val: value to be masked
+ * @hash: value to be masked
  *
  * We use hash values as offset in directories, so values %0 and %1 are
  * reserved for "." and "..". %2 is reserved for "end of readdir" marker. This
-- 
2.55.0

